numpy/distutils/command/build_ext.py

```diff
@@ -262,15 +262,17 @@ def run(self):
         self.build_extensions()
 
         # Copy over any extra DLL files
-        runtime_lib_dir = os.path.join(
-            self.build_lib, self.distribution.get_name(), 'extra-dll')
-        for fn in os.listdir(self.extra_dll_dir):
-            if not fn.lower().endswith('.dll'):
-                continue
-            if not os.path.isdir(runtime_lib_dir):
-                os.makedirs(runtime_lib_dir)
-            runtime_lib = os.path.join(self.extra_dll_dir, fn)
-            copy_file(runtime_lib, runtime_lib_dir)
+        root_pkg = self.distribution.root_pkg
```

Comment:
In which cases would using `root_pkg` not work?

Reply:
Where there isn't a `__config__.py`, which is mainly the f2py tests.

Reply:
How is a root package defined? Can one distribution have multiple root packages?

Reply:
Technically, it can, in which case this scheme would fail. I need some real-world examples in order to figure out how to handle them though. This fixes the example that you mentioned. If you have other examples, please share them.

I'm also going to hint that this is a private attribute because I don't want people relying on this.